frontend/src/components/HomeLoggedIn/HomeLoggedIn.jsx

```diff
@@ -11,10 +13,22 @@ import { generateNetworkConfig } from "utils/NetworkConfig";
 
 function HomeLoggedIn() {
   const [networks, setNetworks] = useState([]);
+  const [, setLoggedIn] = useLocalStorage("loggedIn", false);
+  const [, setDisableAuth] = useLocalStorage("disableAuth", false);
+  const [token, setToken] = useLocalStorage("token", null);
 
   const classes = useStyles();
   const history = useHistory();
 
+  axios.get("/auth/login").then(function (response) {
```

Comment:
Seems to be that this would cause too many redundant requests 🤔

Reply:
There should be another way to fetch. I've tested a bit more and the result does not sound good. On the page load, it could fetch the `/login` endpoint even three or more times (on every HomeLoggedIn component rerender).
![Screenshot 2022-06-12 at 16 18 35](https://user-images.githubusercontent.com/14824142/173235216-14a24257-b736-4c26-9235-9e9b030d5788.png)

Reply:
![Screenshot 2022-06-12 at 16 21 35](https://user-images.githubusercontent.com/14824142/173235270-d6153ba9-4ebb-4893-ba1a-db83d4002a17.png)

Sorry for the late response. The linter should not throw the warring to accept the PR.

Reply:
Would you be fine with removing the code that keeps on checking for the new status and only sets it on first launch? That would remove the redundant request issue but would require the user to clear localStorage if the auth mechanism changes (which shouldn't happen more than once anyway)

Reply:
Seems to be that the necessity to clean localStorage is the best tradeoff. It would be awesome if you could add some info in the docs about setting up the authentication proxy example and adding the notice about the localStorage clean on the ZU_DISABLE_AUTH toggle. 

Thanks!